### ISSUE TITLE
cdn support without exports or require

### ISSUE THREAD

**Issue by francescoagati** [developit/preact-compat#2]
How using preact-compat without exports or require served directly from a cdn?
I have try to embed

    <script src="https://npmcdn.com/preact@2.8.3/dist/preact.js"></script>
    <script src="https://npmcdn.com/preact-compat@0.5.1"></script>

but i get this errors
exports is not defined and require is not defined.
preacts.js works without exports and require but not preact-compat

**Comment by developit** [comment]
Hi there - yes this package is commonjs-only right now. I can put out a nee release and switxh thr export style to use AMD.

For now you might want to use Browserify CDN (http://wzrd.in), which wraps things up for the web:

https://wzrd.in/standalone/preact-compat

**Comment by francescoagati** [comment]
thanks now work. with this configuration
```
    <script src="https://npmcdn.com/preact@2.8.3/dist/preact.js"></script>
    <script src="https://wzrd.in/standalone/preact-compat"></script>

    <script>
      window.React = preactCompat;
      window.ReactDOM = preactCompat;
    </script>

```

**Comment by developit** [comment]
@francescoagati I just published `0.6.0`, which now includes a UMD wrapper. That should mean it works the way you were originally using it from `npmcdn`. Your new configuration will also continue to work the same.
Cheers!

**Comment by francescoagati** [comment]
Thanks to you

**Comment by francescoagati** [comment]
i have try with

    <script src="https://npmcdn.com/preact@2.8.3/dist/preact.js"></script>
     <script src="https://npmcdn.com/preact-compat@0.6.0"></script>

and i get this error

Uncaught ReferenceError: process is not defined preact-compat@0.6.0:1
Uncaught ReferenceError: React is not defined

**Comment by francescoagati** [comment]
instead using the configuration in this file https://github.com/francescoagati/haxe-react-preact-test/blob/master/index.html it works.

   <script src="https://npmcdn.com/preact@2.8.3/dist/preact.js"></script>
    <script src="https://wzrd.in/standalone/preact-compat"></script>

    <script>
      window.React = preactCompat;
      window.ReactDOM = preactCompat;
    </script>

**Comment by developit** [comment]
Just released [0.6.1](https://github.com/developit/preact-compat/releases/tag/0.6.1) which should fix that, sorry!  Here's the demo I am now using to test:
https://jsfiddle.net/developit/61cqu193/

**Comment by francescoagati** [comment]
now works.
 https://github.com/francescoagati/haxe-react-preact-test/blob/223b27ba9f8b44f3d928f18a82f26882cfbf16cd/index.html
Thanks :)
